config/initializers/paper_trail.rb

```diff
@@ -0,0 +1 @@
+PaperTrail.config.track_associations = false
```

Comment:
@tansaku if this is not set it generates an annoying deprecation message, the default in the next version of the gem will default to false, but we need to explicitly tell it `false` now. When I tried `true` our app was crashing, so my guess was that we are not using it, could you corroborate this?